src/components/ppeGuides/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import styled from 'styled-components'
+import { colors } from 'config'
+import { Row } from 'react-bootstrap'
+import DoingBlock from './DoingBlock'
+
+const PPEGuides = (props) => {
```

Comment:
Again can we have props listed out rather than deconstructing, makes it obvious what to expect here.

Reply:
I have change. you can check now